src/MediaWiki/Specials/SpecialDeferredRequestDispatcher.php

```diff
@@ -127,6 +126,16 @@ private function modifyHttpHeader( $header, $message = '' ) {
 		print $message;
 		ob_flush();
 		flush();
+
+		// @see SpecialRunJobs
+		// Once the client receives this response, it can disconnect
+		set_error_handler( function ( $errno, $errstr ) {
```

Comment:
Copied from `SpecialRunJobs`in MW-core because 1.27 throws a bunch of 

```
error] [4eccbce0] /mw-core/index.php/Special:DeferredRequestDispatcher   ErrorException from line 136 of D:\xampp\htdocs\mw-core\includes\WebResponse.php: PHP Warning: Cannot modify header information - headers already sent
#0 [internal function]: MWExceptionHandler::handleError(integer, string, string, integer, array)
#1 [internal function]: setcookie(string, string, integer, string, string, boolean, boolean)
#2 D:\xampp\htdocs\mw-core\includes\WebResponse.php(136): call_user_func(string, string, string, integer, string, string, boolean, boolean)
#3 D:\xampp\htdocs\mw-core\includes\MediaWiki.php(538): WebResponse->setCookie(string, string, integer, array)
#4 D:\xampp\htdocs\mw-core\includes\MediaWiki.php(494): MediaWiki::preOutputCommit(RequestContext)
#5 D:\xampp\htdocs\mw-core\includes\MediaWiki.php(704): MediaWiki->doPreOutputCommit()
#6 D:\xampp\htdocs\mw-core\includes\MediaWiki.php(474): MediaWiki->main()
#7 D:\xampp\htdocs\mw-core\index.php(43): MediaWiki->run()
```

caused/fixed by https://phabricator.wikimedia.org/T115413